modules/transactions/src/main/java/org/apache/ignite/internal/tx/UpdateCommandResult.java

```diff
@@ -94,6 +99,10 @@ public boolean isPrimaryInPeersAndLearners() {
         return primaryInPeersAndLearners;
     }
 
+    public long safeTimestamp() {
```

Comment:
Please add a javadoc explaining what this is

Reply:
🆗